src/main/java/seedu/address/logic/commands/ListTaskCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_TASKS;
+
+import seedu.address.logic.commands.CommandResult.SwitchView;
+import seedu.address.model.Model;
+
+
+/**
+ * Lists all tasks in the address book to the user.
+ */
+public class ListTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "list-task";
```

Comment:
maybe `list-tasks` would be better, just like `list-weddings`

Reply:
Sure, i'll make the change